install_config/configuring_local.adoc

```diff
@@ -364,15 +398,14 @@ to `true`:
 ...
 ----
 
```

Comment:
There is an indent issue for line 360~363.
`volumes(L:360)` should align with `containers(L:352)`, `name(L:363)` should align with `hostPath(L:361)`